org.flameshot.flameshot.yml

```
@@ -0,0 +1,33 @@
+app-id: org.flameshot.flameshot
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: flameshot
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  - --device=dri
+  # Connectivity
+  - --share=network
+  # QtSingleApplication, allow other instances to see log files
+  - --env=TMPDIR=/var/tmp
+  # Allow loading/saving files from anywhere
+  - --filesystem=host
+  # Notification access
+  - --talk-name=org.freedesktop.Notifications
+  # System Tray Icon
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --own-name=org.kde.StatusNotifierItem-2-1
```

```suggestion
  - --own-name=org.kde.*
```
That's the only way to "fix it"

---

Thanks, I will make this change and update.